chunky/src/java/se/llbit/chunky/renderer/RenderWorkerPool.java

```diff
@@ -149,4 +170,42 @@ public void interrupt() {
       worker.interrupt();
     }
   }
+
+  /**
+   * The future to a job. It may be waited on to finish with {@code awaitFinish}.
+   */
+  public static class RenderJobFuture {
```

Comment:
Why not `Future<RenderJob>`?

Reply:
Future would complicate the code a bit more since I would need to implement `cancel` and wait timeouts.